### PR TITLE
Update bank.json

### PR DESCRIPTION
**Pull request by Abdelkrim2003**
adding bank entries and update existense entrie morocco

### CODE REVIEW

**Comment by LaoshuBaby** [comment]
I wander to know why don't you add brand/name for Bank Al-Maghrib, and Credit agricole's.

Also, why don't add name in Crédit du Maroc but add a brand.

Looks like the language selection in this PR have a mess, can you explain the perspestive when you choose to use which language? Thank you!

--
FYI, I found the name guideline of Morocco, but I don't know is those rules suitable for this PR, that need local contibutor(like you)'s opinion.

https://wiki.openstreetmap.org/wiki/Multilingual_names#Morocco

File: data/brands/amenity/bank.json [hunk 1]
```diff
@@ -4544,6 +4556,18 @@
...
     },
+	{
+      "displayName": "Credit agricole (القرض الفلاحي)",
+      "locationSet": {"include": ["ma"]},
+      "tags": {
+        "amenity": "bank",
+        "brand:ar": "القرض الفلاحي المغربي",	
+        "brand": "Credit agricole du Maroc",
+        "brand:wikidata": "Q3006193",
+        "name:fr": "Credit agricole du Maroc"
```
Comment (LaoshuBaby) [inline data/brands/amenity/bank.json:4567]:
```suggestion
        "name:fr": "Credit agricole du Maroc",
```

File: data/brands/amenity/bank.json [hunk 2]
```diff
@@ -8894,13 +8930,16 @@
...
       "id": "ae07f9-11534a",
       "locationSet": {"include": ["ma"]},
       "tags": {
         "amenity": "bank",
-        "brand": "Société Générale سوسيتيه جنرال",
-        "name": "Société Générale سوسيتيه جنرال"
+        "brand": "Société générale Maroc",       
+        "brand:wikidata": "Q3488352",  
+        "brand:ar": "الشركة العامة",       
+        "name:fr": "Société générale Maroc"
```
Comment (LaoshuBaby) [inline data/brands/amenity/bank.json:8941]:
```suggestion
        "name:fr": "Société générale Maroc",
```

**Comment by Abdelkrim2003** [comment]
if the languages are the issue,we can remove them

**Comment by LaoshuBaby** [comment]
> we can remove them

No, I means, looks like Morocco's tagging guideline ask for both `ar` and `fr` (also, English is a good fallback language), but you sometimes choose not to include `ar`, sometimes choose not to include `fr`, different entries choose to include different languages, but them are in the same country, and I'm confused by this inconsistency.

For example: Hong Kong entry selection contains `name`+`name:zh`+`name:en`, because those 2 are both official language

**Comment by Abdelkrim2003** [comment]
In Morocco they have 2 official languages(arabic,tamazight) and french is used for adminstration purpose,
but most of the time only arabic and french is used(example: streetnames, are only in arabic and french,in my  earlier contribution about bus network they use only french and sometimes arabic, thats also cofusing for me what language to add

**Comment by LaoshuBaby** [comment]
> thats also cofusing for me what language to add

OK, so I add brand according to other languages you offer, I choose the English/French one.

You can see if this is accaptable to Morocoo's local community. (I can't found any contact way of Morocco, the mailing list looks like haven't active for many years, I sent a email to [ym-Universit-Mohammed-V-Rabat](https://github.com/osmlab/osm-community-index/blob/69e3b87e901e9487c6f7e7c0887263a7e3894be9/resources/africa/morocco/ym-Universit-Mohammed-V-Rabat.json))

**Comment by Abdelkrim2003** [comment]
I created all the guidelinepage on wiki openstreetmap morocco myself,in the beggining it was a mess mapping morocco,the Morocco's local community does not even exist anymore but most of the contributors follows the guideline so only arabic,and french is used and tamazight also if names are known

**Comment by LaoshuBaby** [comment]
Thanks for your reply, now I changed the `brand`/`name`'s value follow the guideline.

**Pull request merged**